zio-http/src/main/scala/zhttp/service/server/ServerChannelInitializer.scala

```diff
@@ -46,6 +46,16 @@ final case class ServerChannelInitializer[R](
     if (cfg.requestDecompression._1)
       pipeline.addLast(HTTP_REQUEST_DECOMPRESSION, new HttpContentDecompressor(cfg.requestDecompression._2))
 
+    // HttpContentCompressor
+    if (cfg.responseCompression._2.nonEmpty)
+      pipeline.addLast(
+        HTTP_RESPONSE_COMPRESSION,
+        new HttpContentCompressor(
+          cfg.responseCompression._1,
+          cfg.responseCompression._2.map(_.toJava): _*,
```

Comment:
This splat(_*) here is necessary because `HttpContentCompressor` takes in a variadic arg for CompressionOptions.